.circleci/config.yml

```diff
@@ -568,6 +574,23 @@ jobs:
             bundle exec fastlane android create_snapshots_repo_pr
       - android/save-build-cache
 
+  run-revenuecatui-ui-tests:
+    description: "Run RevenueCatUI UI tests for Android in CircleCI"
+    <<: *android-machine-emulator
+    steps:
+      - checkout
+      - install-sdkman
+      - android/create-avd:
+          avd-name: test-revenuecat-ui
+          system-image: system-images;android-32;google_apis;x86_64
```

Comment:
What do you think (for a separate PR), to add a document like https://github.com/RevenueCat/purchases-ios/blob/main/Contributing/UpdatingRequirements.md to list all the different places that need to be updated when upgrading Android, etc?
I feel like this one in particular would be easy to miss. I created that for iOS because I realized that these were all spread out in many places.

Reply:
That's a great idea! I added a linear ticket to keep track of this 👍